From 0458c7723fe8da9803b6e363ef4d5c9348cdf282 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 16 Jul 2024 08:16:50 +0000
Subject: [PATCH] Removed deprecated HAL renderscript

Renderscript hal has been deprecated in A15.
Removing it.

Tracked-On: OAM-122201
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/framework_manifest.xml | 9 ---------
 groups/device-specific/caas/manifest.xml           | 9 ---------
 groups/device-specific/caas/product.mk             | 1 -
 3 files changed, 19 deletions(-)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index f904fe2..2132b97 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -127,15 +127,6 @@
         <name>android.hardware.camera.provider</name>
 	<fqname>ICameraProvider/external/0</fqname>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.renderscript</name>
-        <transport arch="32+64">passthrough</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IDevice</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <!--vendor-ndk>
         <version>34</version>
     </vendor-ndk-->
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index cdd3a72..330814a 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -53,15 +53,6 @@
         <name>android.hardware.camera.provider</name>
 	<fqname>ICameraProvider/external/0</fqname>
     </hal>
-    <hal format="hidl">
-        <name>android.hardware.renderscript</name>
-        <transport arch="32+64">passthrough</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IDevice</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <!--<kernel target-level="5"/>-->
     <!--sepolicy>
         <version>34.0</version>
diff --git a/groups/device-specific/caas/product.mk b/groups/device-specific/caas/product.mk
index 2c1dfe9..6dc8986 100755
--- a/groups/device-specific/caas/product.mk
+++ b/groups/device-specific/caas/product.mk
@@ -27,7 +27,6 @@ PRODUCT_PACKAGES +=  \
                     camera.device@1.0-impl \
                     android.hardware.camera.provider@2.4-impl \
                     android.hardware.graphics.allocator-service.minigbm \
-                    android.hardware.renderscript@1.0-impl \
                     android.hardware.identity \
                     android.hardware.identity-service.example \
 		    android.hardware.graphics.composer3-service.intel \
-- 
2.45.2

